bot/cogs/snekbox.py

```diff
@@ -292,6 +298,18 @@ async def eval_command(self, ctx: Context, *, code: str = None) -> None:
             await ctx.invoke(self.bot.get_command("help"), "eval")
             return
 
+        if Roles.helpers in (role.id for role in ctx.author.roles):
+            self.bot.stats.incr("snekbox_usages.roles.helpers")
+        else:
+            self.bot.stats.incr("snekbox_usages.roles.developers")
+
+        if ctx.channel.category_id == Categories.help_in_use:
```

Comment:
We don't allow this command in DMs, right? Cause if `channel` is a `DMChannel`, then `category_id` will be an `AttributeError`.

I want to note I have a PR which adds a utility function for checking help channels. It's not done yet, but when it is, this should be replaced with it. https://github.com/python-discord/bot/blob/4378e1f12907a73a26e56760c4a5717ff2a017d2/bot/utils/channel.py#L10

Reply:
Yes, this have guild only decorator.